### DISCUSSION TITLE
Does gunicorn want to support WSGI+ ?

### DISCUSSION THREAD

**Discussion by pwtail** [benoitc/gunicorn#3443] [Ideas]
Here is the proposal https://github.com/pwtail/wsgi_plus

Here is the proof of concept https://github.com/pwtail/gunicorn/pull/1/files#diff-9818e6c0e3d6054dc383f77ce881ba79f8090a904fb3abd9892306f096e58319

**Comment by benoitc** [comment]
where is discussed this specification ?

**Comment by benoitc** [comment]
@pwtail bump

**Comment by benoitc** [comment]
Interesting approach - allows sync apps to yield `Future` objects during I/O without full async rewrite.

Is there still interest? Any real use cases where this would be preferred over ASGI migration?
